src/config.js

```diff
@@ -63,8 +65,30 @@ function debug () {
   if (HEROKU_DEBUG) return parseInt(HEROKU_DEBUG)
   return 0
 }
+function skipAnalytics (userConfig: UserConfig) {
+  if (userConfig.skipAnalytics) {
+    return true
+  } else if (process.env['TESTING'] === '1') {
+    return true
+  }
+  return false
+}
+
+let loadUserConfig = function (configDir) {
+  try {
+    const config: UserConfig = fs.readJsonSync(path.join(configDir, 'user_config.json'), 'utf8')
```

Comment:
@KarateCowboy It is not what we called the file, it was `config.json` https://github.com/heroku/cli/blob/master/config.go#L19